prime-angular-frontend/src/app/modules/site-registration/pages/remote-users-page/remote-users-page.component.html

```diff
@@ -24,6 +24,21 @@
 
   </form>
 
+  <div class="mb-4"
+       *ngIf="addedFirstRemoteUser">
+    <app-alert type="success"
+               icon="check_circle_outline">
+      <ng-container #alertTitle
+                    class="alert-title">
+        Remote Practitioner added
```

Comment:
I can add a user and remove then untoggle and the banner keeps saying that a remote practitioner was added
Is this expected?
![image](https://user-images.githubusercontent.com/34261366/145286267-5ece5085-0d45-44a9-a4cb-335521c7ceb8.png)